[2025-01-06T17:37:08.217+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T17:37:08.249+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.end_task manual__2025-01-06T17:36:52.905070+00:00 [queued]>
[2025-01-06T17:37:08.271+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.end_task manual__2025-01-06T17:36:52.905070+00:00 [queued]>
[2025-01-06T17:37:08.272+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T17:37:08.298+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): end_task> on 2025-01-06 17:36:52.905070+00:00
[2025-01-06T17:37:08.306+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=150) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T17:37:08.309+0000] {standard_task_runner.py:72} INFO - Started process 151 to run task
[2025-01-06T17:37:08.310+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'end_task', 'manual__2025-01-06T17:36:52.905070+00:00', '--job-id', '852', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmp86vi5ym2']
[2025-01-06T17:37:08.313+0000] {standard_task_runner.py:105} INFO - Job 852: Subtask end_task
[2025-01-06T17:37:08.390+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.end_task manual__2025-01-06T17:36:52.905070+00:00 [running]> on host f560492d2669
[2025-01-06T17:37:08.547+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T17:36:52.905070+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T17:36:52.905070+00:00'
[2025-01-06T17:37:08.549+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T17:37:08.583+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-06T17:37:08.585+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here we end! "']
[2025-01-06T17:37:08.602+0000] {subprocess.py:99} INFO - Output:
[2025-01-06T17:37:08.605+0000] {subprocess.py:106} INFO - Here we end!
[2025-01-06T17:37:08.606+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-06T17:37:08.661+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T17:37:08.663+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=end_task, run_id=manual__2025-01-06T17:36:52.905070+00:00, execution_date=20250106T173652, start_date=20250106T173708, end_date=20250106T173708
[2025-01-06T17:37:08.726+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T17:37:08.768+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-06T17:37:08.774+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
